spec/rspec/expectations/extensions/kernel_spec.rb

```diff
@@ -19,7 +19,7 @@
     }.to fail_with("the failure message")
```

Comment:
Those examples don't belong to `extensions/kernel` here since we don't patch `Kernel`/`Object`.
Also, we describe `Object` and removed methods.

```ruby
RSpec.describe Object, "#should" do
RSpec.describe Object, "#should_not" do
```

I'd love to move this somewhere. What would you recommend, @JonRowe ?

Reply:
The remaining `"#should"` / `"#should_not"` context specs seem to be about the protocol, a bunch of those are in `spec/rspec/expectations/syntax_spec.rb` maybe that would be a good place for those? WDYT?

The basic object ones were a should regression check when written, and could arguably be removed

Reply:
It seems that it's extensively covered in `spec/rspec/matchers/dsl_spec.rb`. I'll just remove this.